code/game/gamemodes/clock_cult/clock_scriptures/scripture_drivers.dm

```diff
@@ -231,6 +231,13 @@
 	quickbind = TRUE
 	quickbind_desc = "Creates a stargazer, which generates power when near starlight."
 
+/datum/clockwork_scripture/create_object/stargazer/check_special_requirements()
+	var/area/A = get_area(invoker)
+	if(A.outdoors || A.map_name == "Space")
```

Comment:
ew, don't check strings like this, it's bad and might not catch stuff properly.

Reply:
There doesn't seem to be a better way; if there is, it's a proc that I'm unaware of. The only thing I can think of is adding a new helper proc that does the same thing like `initial_space_area(turf)` or something that checks if the area is in space or was initially space.

Reply:
Xhuis I told you about the blob proc, I used it for the narsie rune too before I limited it to a randomized set of rooms.

Reply:
I did actually search for that, but I couldn't locate it in the code. If you know the syntax for it I'd be happy to include it.